### PR TITLE
Doc/Wadl Updates

### PR DESCRIPTION
**Pull request by jcru**
- Mgmt-host-update added to docs/wadl & example generator
- Mgmt-instance-hwinfo added to to docs/wadl & example generator
- Added tenant_id field to sample responses for mgmt-host-list-all-instances

### CODE REVIEW

**Comment by cp16net** [comment]
did you generate the examples or do it by hand?

File: apidocs/src/resources/samples/db-mgmt-get-host-detail-response.json [hunk 1]
```diff
@@ -10,13 +10,15 @@ Date: Tue, 24 Jul 2012 19:48:33 GMT
                 "id": "6e038c18-902f-4a41-b2a2-5f46fa62fc1b", 
                 "name": "json_rack_instance", 
                 "server_id": "64438fb6-4b98-4bd8-975e-222bc11daf74", 
-                "status": "ACTIVE"
+                "status": "ACTIVE",
+		"tenant_id": "1234"
```
Comment (cp16net) [inline apidocs/src/resources/samples/db-mgmt-get-host-detail-response.json:14]: this looks hand done? should be spaced over like above?

File: apidocs/src/resources/samples/db-mgmt-instance-hwinfo-response.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+Content-Length: 48
+Date: Tue, 24 Jul 2012 19:48:33 GMT
+
+{
+    "hwinfo": {
+        "mem_total": 524288,
```
Comment (ed-) [inline apidocs/src/resources/samples/db-mgmt-instance-hwinfo-response.json:8]: ### THAT'S A LOT OF MEMS!

Comment (TimSimpsonR) [inline apidocs/src/resources/samples/db-mgmt-instance-hwinfo-response.json:8]: Thank you my friend. Here is your reward...

**Pull request closed without merging**